src/container/navigation/navigation.jsx

```diff
@@ -0,0 +1,39 @@
+import experiosLogo from 'assets/logo/logo1.png';
```

Comment:
inconsistent file naming
Pascal for scss file
camel for jsx file

Reply:
I think you meant pascal for jsx and camel for scss. Right?